lib/src/main/java/uk/gov/di/ipv/cri/passport/library/domain/CredentialSubject.java

```diff
@@ -0,0 +1,128 @@
+package uk.gov.di.ipv.cri.passport.library.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+public class CredentialSubject {
+
+    @JsonProperty private final Name name;
+    @JsonProperty private final String passportNumber;
+
+    @JsonProperty
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    private final LocalDate birthDate;
```

Comment:
Does this var type now need to be the new BirthDate class

Reply:
Done